snippets/csharp/VS_Snippets_Wpf/DrawingMiscSnippets_snip/CSharp/EnumerateDrawingsExample.xaml.cs

```diff
@@ -5,65 +5,59 @@
 using System.Windows.Media.Animation;
 using System.Windows.Navigation;
 
-
 namespace SDKSample
 {
-
-
-    public partial class EnumerateDrawingsExample: Page
+    public partial class EnumerateDrawingsExample : Page
     {
-
         public EnumerateDrawingsExample()
         {
             InitializeComponent();
         }
 
-        private void retrieveDrawings(object sender, RoutedEventArgs e)
+        private void EnumerateButton_OnClick(object sender, RoutedEventArgs e)
         {
             RetrieveDrawing(MainBorder);
         }
 
         //<SnippetGraphicsMMRetrieveDrawings>
         public void RetrieveDrawing(Visual v)
         {
-            DrawingGroup dGroup = VisualTreeHelper.GetDrawing(v);
-            EnumDrawingGroup(dGroup);
-
+            DrawingGroup drawingGroup = VisualTreeHelper.GetDrawing(v);
+            EnumDrawingGroup(drawingGroup);
         }
 
-         // Enumerate the drawings in the DrawingGroup.
-         public void EnumDrawingGroup(DrawingGroup drawingGroup)
-         {
-             DrawingCollection dc = drawingGroup.Children;
-
-             // Enumerate the drawings in the DrawingCollection.
-             foreach (Drawing drawing in dc)
-             {
-                 // If the drawing is a DrawingGroup, call the function recursively.
-                 if (drawing.GetType() == typeof(DrawingGroup))
-                 {
-                     EnumDrawingGroup((DrawingGroup)drawing);
-                 }
-                 else if (drawing.GetType() == typeof(GeometryDrawing))
-                 {
-                     // Perform action based on drawing type.  
-                 }
-                 else if (drawing.GetType() == typeof(ImageDrawing))
-                 {
-                     // Perform action based on drawing type.
-                 }
-                 else if (drawing.GetType() == typeof(GlyphRunDrawing))
-                 {
-                     // Perform action based on drawing type.
-                 }
-                 else if (drawing.GetType() == typeof(VideoDrawing))
-                 {
-                     // Perform action based on drawing type.
-                 }
-             }
-         }
-         //</SnippetGraphicsMMRetrieveDrawings>
-     }
-    
+        // Enumerate the drawings in the DrawingGroup.
+        public void EnumDrawingGroup(DrawingGroup drawingGroup)
+        {
+            DrawingCollection dc = drawingGroup.Children;
+
+            // Enumerate the drawings in the DrawingCollection.
+            foreach (Drawing drawing in dc)
+            {
+                // If the drawing is a DrawingGroup, call the function recursively.
+                if (drawing is DrawingGroup @group)
```

Comment:
I'm curious why you use `@group` instead of `group`

Reply:
@BillWagner Thx for your review